packages/@aws-cdk/aws-codepipeline-actions/test/integ.pipeline-code-build-multiple-inputs-outputs.ts

```diff
@@ -65,6 +65,7 @@ const testAction = new cpactions.CodeBuildAction({
   type: cpactions.CodeBuildActionType.TEST,
   actionName: 'Build2',
   project,
+  executeBatchBuild: true,
```

Comment:
Let's leave this unchanged. If you really want to, you can add a new integ test just for this, but I won't require it to get the PR merged.